mappings/net/minecraft/util/Util.mapping

```diff
@@ -7,6 +7,9 @@ CLASS net/minecraft/class_156 net/minecraft/util/Util
 	FIELD field_24622 BOOTSTRAP_EXECUTOR Ljava/util/concurrent/ExecutorService;
 	FIELD field_25140 NIL_UUID Ljava/util/UUID;
 	FIELD field_33859 JAR_FILE_SYSTEM_PROVIDER Ljava/nio/file/spi/FileSystemProvider;
+	FIELD field_34891 MAX_BACKGROUND_THREADS I
```

Comment:
Maybe `MAX_PARALLELISM` which is used by java's documentation: https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/concurrent/ForkJoinPool.html#%3Cinit%3E(int)

```suggestion
	FIELD field_34891 MAX_PARALLELISM I
```

Your current name is wrong; it should be `MAX_MAX_BACKGROUND_THREADS`, but that corrected name sounds a bit weird. Hence this alternative.